src/views/forms/FormTransactionBase/FormTransactionBase.ts

```diff
@@ -327,31 +340,185 @@ export class FormTransactionBase extends Vue {
    * Process form input
    * @return {void}
    */
+  public currentAccount: AccountsModel
+  public currentPeer: Record<string, any>
+  public signedTransactions: SignedTransaction[]
+
   public async onSubmit() {
     const transactions = this.getTransactions()
 
     this.$store.dispatch('diagnostic/ADD_DEBUG', `Adding ${transactions.length} transaction(s) to stage (prepared & unsigned)`)
+    const currentFee = transactions.map(t=>{
+      return t.maxFee
+    } )
 
     // - check whether transactions must be aggregated
     // - also set isMultisig flag in case of cosignatory mode
     if (this.isAggregateMode()) {
       this.$store.commit('wallet/stageOptions', {
         isAggregate: true,
         isMultisig: this.isMultisigMode(),
+        
       })
-    }
-
-    // - add transactions to stage (to be signed)
+    }     
+  const options = this.$store.getters['wallet/stageOptions']
+  //check isLedger wallet
+  if(this.currentWallet.values.get('type')== WalletType.fromDescriptor('Ledger')){
+    this.$Notice.success({
+      title: this['$t']('Verify information in your device!') + ''
+    })
+    const nodeUrl = Object.values(this.currentPeer)[3]
+    const signer = this.currentSigner.publicKey;//accountResult.publicKey;
+    const transport = await TransportWebUSB.create();
+    const symbolLedger = new SymbolLedger(transport, "XYM");
+    const currentPath = this.currentWallet.values.get('path')
+    const accountIndex = Number(currentPath.substring(currentPath.length-2,currentPath.length-1))
+    const networkType =  Number(currentPath.substring(currentPath.length-10,currentPath.length-7))
+    const accountResult = await symbolLedger.getAccount(`m/44'/43'/${networkType}'/0'/${accountIndex}'`)
+    const { address, publicKey, path } = accountResult;
+    const generationHash = this.$store.getters['network/generationHash']
+    // const addr = Address.createFromPublicKey(signer,this.networkType)
+    try {
+        //Get account from ledger.
+        const signedTransactions =[]
+        
+        
+        if (options.isAggregate ){
+          if(!options.isMultisig){ //Create New Mosaics
```

Comment:
misleading comment ; this method is not used only with mosaic definition, its used for all transactions. Please, do not modify the aggregate/multisig transaction creation process.